addons/mail/static/src/core/common/message.xml

```diff
@@ -82,7 +82,7 @@
                                                             'mb-0 py-2': !message.is_note,
                                                             'o-note': message.is_note,
                                                             'align-self-start rounded-end-3 rounded-bottom-3': !state.isEditing and !message.is_note,
-                                                            'o-mail-Message-editable flex-grow-1': state.isEditing,
```

Comment:
unnecessary remove, especially in stable?

Reply:
I removed it so that we don't have cryptic classname to tell this is editing, whereas top-level classname modifier `o-editing` is clearer and reusable.